lib/assets/news-stylesheet.scss

```diff
@@ -94,42 +96,66 @@ h1, h2, h3, h4, h5, h6 {
     color: $primary-dark-color;
     font-family: $title-font;
     font-weight: bold;
-}
-
-h1, h2, h3, h4 {
-    line-height: 1.18;
+    line-height: 1.25;
+    margin-top: 3.0rem;
+    margin-bottom: 0.5rem;
 }
 
 h1 {
     font-size: 2.8rem;
+    + h2, + h3, + h4, + h5, + h6 {
+        margin-top: 0;
+    }
+}
+
+h1:first-child {
+    margin-top: 1rem;
 }
 
 h2 {
     font-size: 2.2rem;
+    + h3, + h4, + h5, + h6 {
+        margin-top: 0;
+    }
 }
 
 h3 {
     font-size: 1.7rem;
+    + h4, + h5, + h6 {
+        margin-top: 0;
+    }
 }
 
 h4 {
     font-size: 1.4rem;
+    margin-bottom: 0.5rem;
+    + h5, + h6 {
+        margin-top: 0;
+    }
 }
 
 h5 {
     font-size: 1.2rem;
-    line-height: 1.333;
+    margin-bottom: 0.25rem;
+    + h6 {
+        margin-top: 0;
+    }
 }
 
 h6 {
-    font-style: italic;
+    font-size: 1rem;
+    line-height: $body-line-height;
     font-weight: normal;
+    font-style: italic;
 }
 
 hr {
-    border-color: $background-light-color;
-    border-width: 1px 0 0 0;
-    margin-bottom: 2rem;
+    display: block;
+    height: 1px;
+    border: 0;
+    border-top: 1px solid $background-light-color;
+    padding: 0;
+    margin: 3rem 0 2rem;
```

Comment:
The left and right margins are set in `responsive-elements` so it would be better to specify only `margin-top` and `margin-bottom` here.